tests/test_tools.py

```diff
@@ -218,13 +218,12 @@ def test_subdomains(self):
         assert openreview.tools.subdomains('   ') == []
 
     def test_replace_members_with_ids(self, client, test_client):
-
         posted_group = client.post_group(openreview.Group(id='test.org',
             readers=['everyone'],
             writers=['~Super_User1'],
             signatures=['~Super_User1'],
             signatories=['~Super_User1'],
-            members=['test@mail.com', '~SomeFirstName_User1', '~Another_Name1']
+            members=['test@mail.com', '~SomeFirstName_User1', '~Another_Name1', 'NewGroup']
```

Comment:
if the group has a member with a tilde id where the number is missing, the member is removed from the group and prints this:

`Invalid profile/group id in group aclweb.org/ACL/ARR/2021/December/Paper1/Reviewers : ~Melisa_Bok`

I would like to keep ~Melisa_Bok in the group, the replace function should only replace, don't remove members.

Reply:
wouldn't this cause problems later on?